src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -16,6 +16,8 @@ import UploadDetachedFileValidation from './UploadDetachedFileValidation.jsx';
 
```

Comment:
I tried removing all the showMeta stuff and replaced with checking for the props and it didn't work